### DISCUSSION TITLE
LaTeX inlline math not rendered in the mobile app

### DISCUSSION THREAD

**Discussion by rigobert9** [community/community#31878] [Mobile] [answered]
Hi !
I'm using markdown notes with LaTeX math for my notes, and I sometime need to look them up on mobile. Unfortunately, $$ formulas aren't rendered in markdown files. Is there any plan of adding this feature to the mobile app ?

**Comment by rigobert9** [comment, marked as answer]
My bad, didn't look up other questions well enough, could someone close this discussion please ? I'm very sorry

**Reply by adamjstewart** [reply]
Is this issue being discussed in another question? If so, can someone share the link to that discussion? I couldn't find it from a quick search.
